src/triangulum/config.clj

```diff
@@ -85,16 +100,65 @@
    (reset! config-file new-config-file)
    (reset! config-cache (read-config @config-file))))
 
+(defn namespaced-config?
+  "Returns true if the given configuration map is namespaced, otherwise false."
+  [config]
+  (s/valid? ::namespaced-config config))
+
+(defn nested-config?
+  [config]
+  (not (namespaced-config? config)))
+
+(defn split-ns-key
+  "Given a namespaced key, returns a vector of unnamespaced keys."
+  [ns-key]
+  [(get-mapped-key-ns ns-key) (-> ns-key (name) (keyword))])
+
+(defn join-un-key
+  "Given a sequence of unnamespaced keys, returns a single namespaced key."
+  ([key-path] (join-un-key key-path {:prefix "triangulum"}))
+  ([key-path {:keys [prefix]}]
+   (let [[n k] key-path
+         n (or (get un->ns-mapping n) n)]
```

Comment:
Here we can remove the `or` function call, passing the default value to the `get` function.